Makefile

```diff
@@ -1,6 +1,6 @@
 CC = /usr/bin/gcc
-CFLAGS = -Wall -g -O3 -Wextra -Wpedantic
-LDLIBS = -lcrypto
+CFLAGS = -Wall -g -O3 -Wextra -Wpedantic #-fsanitize-address-use-after-return=always -fsanitize=address
+LDLIBS = -lcrypto -I/opt/homebrew/opt/openssl@3/include -L/opt/homebrew/opt/openssl@3/lib
```

Comment:
I'm not sure `-I` belongs in `LDLIBS`.

Reply:
This is specific for Mac only. I will remove them on in final version.